### ISSUE TITLE
Reproducible output jar

### ISSUE THREAD

**Issue by satsen** [Guardsquare/proguard#414]
Is it possible to disable the preservation of file last modified timestamps in the output JAR so that everything is just 0? For example like you can do with the Gradle archival option "preserveFileTimestamps".

Additionally, is it possible to use a "[reproducible file order](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.Jar.html#org.gradle.api.tasks.bundling.Jar:reproducibleFileOrder)"?

**Comment by ianbrandt** [comment]
This may be a duplicate of #28.

**Comment by satsen** [comment]
@ianbrandt You're right, thanks. Even though my issue is not about Gradle specifically I'm closing this one since I believe if they fix #28 they would implement support for it in general.
